Index: src/templates/blogTemplate.js
===================================================================
diff --git a/src/templates/blogTemplate.js b/src/templates/blogTemplate.js
deleted file mode 100644
--- a/src/templates/blogTemplate.js	(revision 567bead0fff14b18679aa9cead70c0b7085d46d0)
+++ /dev/null	(revision 567bead0fff14b18679aa9cead70c0b7085d46d0)
@@ -1,30 +0,0 @@
-import React from 'react'
-import { graphql } from 'gatsby'
-
-export default function Template({ data }) {
-  console.log(data)
-  const { markdownRemark } = data
-  const { frontmatter, html } = markdownRemark
-  return (
-    <div className="blog-post-container">
-      <div className="blog-post">
-        <h1>{frontmatter.title}</h1>
-        <h2>{frontmatter.date}</h2>
-        <div className="blog-post-content" dangerouslySetInnerHTML={{ __html: html }} />
-      </div>
-    </div>
-  )
-}
-
-export const pageQuery = graphql`
-  query($slug: String!) {
-      markdownRemark(frontmatter: { slug: { eq: $slug } }) {
-          html
-          frontmatter {
-              date(formatString: "MMMM DD, YYYY")
-              slug
-              title
-          }
-      }
-  }
-`
\ No newline at end of file
Index: gatsby-node.js
===================================================================
diff --git a/gatsby-node.js b/gatsby-node.js
deleted file mode 100644
--- a/gatsby-node.js	(revision 567bead0fff14b18679aa9cead70c0b7085d46d0)
+++ /dev/null	(revision 567bead0fff14b18679aa9cead70c0b7085d46d0)
@@ -1,38 +0,0 @@
-exports.createPages = async ({ actions, graphql, reporter }) => {
-  const { createPage } = actions
-  const blogPostTemplate = require.resolve(`./src/templates/blogTemplate.js`)
-
-  const result = await graphql(`
-    {
-      allMarkdownRemark(
-        sort: { order: DESC, fields: [frontmatter___date] }
-        limit: 1000
-      ) {
-        edges {
-          node {
-            frontmatter {
-              slug
-            }
-          }
-        }
-      }
-    }
-  `)
-
-  if (result.errors) {
-    reporter.panicOnBuild(`Error while running GraphQL query.`)
-    return
-  }
-
-  result.data.allMarkdownRemark.edges.forEach(({ node }) => {
-    const slug = node.frontmatter.slug
-    console.log(slug)
-    createPage({
-      path: slug,
-      component: blogPostTemplate,
-      context: {
-        slug: slug
-      }
-    })
-  })
-}
\ No newline at end of file
Index: src/blog/2020-12-28-test-two.md
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>---\r\nslug: \"{{slug}}\"\r\ntitle: Test Two\r\ndate: 2020-12-28T10:28:44.796Z\r\nthumbnail: /images/uploads/secrets.jpg\r\n---\r\nTest test.
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/blog/2020-12-28-test-two.md b/src/blog/2020-12-28-test-two.md
--- a/src/blog/2020-12-28-test-two.md	(revision 567bead0fff14b18679aa9cead70c0b7085d46d0)
+++ b/src/blog/2020-12-28-test-two.md	(date 1609152560089)
@@ -1,5 +1,4 @@
 ---
-slug: "{{slug}}"
 title: Test Two
 date: 2020-12-28T10:28:44.796Z
 thumbnail: /images/uploads/secrets.jpg
Index: src/blog/2020-12-26-this-is-a-test.md
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>---\r\ntitle: This is a test\r\nslug: blog/2020-12-26-this-is-a-test\r\ndate: 2020-12-26T10:35:16.937Z\r\nthumbnail: /images/uploads/secrets.jpg\r\n---\r\nThis is to test the CMS functionality of this Gatsby generated site. Will it work? Who knows!
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/blog/2020-12-26-this-is-a-test.md b/src/blog/2020-12-26-this-is-a-test.md
--- a/src/blog/2020-12-26-this-is-a-test.md	(revision 567bead0fff14b18679aa9cead70c0b7085d46d0)
+++ b/src/blog/2020-12-26-this-is-a-test.md	(date 1609152560066)
@@ -1,6 +1,5 @@
 ---
 title: This is a test
-slug: blog/2020-12-26-this-is-a-test
 date: 2020-12-26T10:35:16.937Z
 thumbnail: /images/uploads/secrets.jpg
 ---
Index: static/admin/config.yml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>backend:\r\n  name: git-gateway\r\n  branch: master\r\n\r\npublish_mode: editorial_workflow\r\nmedia_folder: \"static/images/uploads\"\r\npublic_folder: \"/images/uploads\"\r\n\r\ncollections:\r\n  - name: \"blog\"\r\n    label: \"Blog\"\r\n    folder: \"src/blog\"\r\n    create: true\r\n    slug: \"{{year}}-{{month}}-{{day}}-{{slug}}\" # Filename template, e.g., YYYY-MM-DD-title.md\r\n    fields: # The fields for each document, usually in front matter\r\n      - { label: \"Layout\", name: \"slug\", widget: \"hidden\", default: \"{{slug}}\" }\r\n      - { label: \"Title\", name: \"title\", widget: \"string\" }\r\n      - { label: \"Publish Date\", name: \"date\", widget: \"datetime\" }\r\n      - { label: \"Featured Image\", name: \"thumbnail\", widget: \"image\" }\r\n      - { label: \"Body\", name: \"body\", widget: \"markdown\" }
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/static/admin/config.yml b/static/admin/config.yml
--- a/static/admin/config.yml	(revision 567bead0fff14b18679aa9cead70c0b7085d46d0)
+++ b/static/admin/config.yml	(date 1609152775450)
@@ -7,14 +7,26 @@
 public_folder: "/images/uploads"
 
 collections:
-  - name: "blog"
-    label: "Blog"
+  - name: "list"
+    label: "List"
     folder: "src/blog"
     create: true
-    slug: "{{year}}-{{month}}-{{day}}-{{slug}}" # Filename template, e.g., YYYY-MM-DD-title.md
-    fields: # The fields for each document, usually in front matter
-      - { label: "Layout", name: "slug", widget: "hidden", default: "{{slug}}" }
-      - { label: "Title", name: "title", widget: "string" }
-      - { label: "Publish Date", name: "date", widget: "datetime" }
-      - { label: "Featured Image", name: "thumbnail", widget: "image" }
-      - { label: "Body", name: "body", widget: "markdown" }
\ No newline at end of file
+    slug: "{{year}}-{{month}}-{{day}}-{{slug}}"
+    fields:
+      - {label: "Template Key", name: "templateKey", widget: "hidden", default: "list-page"}
+      - {label: "Title", name: "title", widget: "string"}
+      - {label: "Description", name: "description", widget: "text"}
+      - {label: "Body", name: "body", widget: "markdown"}
+      - {label: "Tags", name: "tags", widget: "list", required: false}
+    editor: false
+
+  - name: "pages"
+    label: "Pages"
+    files:
+      - file: "src/pages/lists/index.md"
+        label: "Lists"
+        name: "lists"
+        fields:
+          - {label: "Template Key", name: "templateKey", widget: "hidden", default: "lists-page"}
+          - {label: "Title", name: "title", widget: "string"}
+          - {label: "Body", name: "body", widget: "markdown"}
\ No newline at end of file
